From 5cbb5c5f9458963a7229b463be2ad5d1ee398058 Mon Sep 17 00:00:00 2001
From: Alexander Martinz <amartinz@shiftphones.com>
Date: Tue, 19 Mar 2019 18:58:11 +0100
Subject: [PATCH 4/6] LightSettingsDialog: add bundle extras for preview color
 and duration

The sdk now accepts an abstract way to force preview values.

Change-Id: I8ca686ab455e19b928b277585c46099c3ac6daf9
Signed-off-by: Alexander Martinz <amartinz@shiftphones.com>
---
 .../lineageparts/notificationlight/LightSettingsDialog.java    | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/org/lineageos/lineageparts/notificationlight/LightSettingsDialog.java b/src/org/lineageos/lineageparts/notificationlight/LightSettingsDialog.java
index ba6c6b8..dd24e1c 100644
--- a/src/org/lineageos/lineageparts/notificationlight/LightSettingsDialog.java
+++ b/src/org/lineageos/lineageparts/notificationlight/LightSettingsDialog.java
@@ -318,6 +318,9 @@ public class LightSettingsDialog extends AlertDialog implements
         if  (mLedBrightness > 0 && mLedBrightness < LedValues.LIGHT_BRIGHTNESS_MAXIMUM) {
             b.putInt(LineageNotification.EXTRA_FORCE_LIGHT_BRIGHTNESS, mLedBrightness);
         }
+        b.putInt(LineageNotification.EXTRA_FORCE_PREVIEW_COLOR, color);
+        b.putInt(LineageNotification.EXTRA_FORCE_PREVIEW_LIGHT_ON_MS, speedOn);
+        b.putInt(LineageNotification.EXTRA_FORCE_PREVIEW_LIGHT_OFF_MS, speedOff);
 
         createNotificationChannel();
 
-- 
2.17.1

